modules/claims_api/lib/claims_api/evss_bgs_mapper.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class EvssBgsMapper
+    attr_reader :evss_id, :list_data
+
+    def initialize(claim)
+      @data = add_claim(claim)
+      @list_data = {}
+    end
+
+    def add_claim(claim)
+      @data = {}
+      @data.merge!(claim)
+      @data.deep_stringify_keys
+    end
+
+    def map_and_build_object
+      claim = EVSSClaim.new
+      claim['data'] = @data
+
+      claim['evss_id'] = @data['benefit_claim_id']
+      claim['data']['contention_list'] = @data['contentions'] || []
+      claim['data']['va_representative'] = @data['poa']
+      claim['data']['development_letter_sent'] = @data['development_letter_sent']
+      claim['data']['decision_letter_sent'] = @data['decision_notification_sent']
+      claim['data']['documents_needed'] = @data['wwsnfy'].present?
+      claim['data']['waiver5103_submitted'] = @data['filed5103_waiver_ind'] == 'Y'
+      claim['data']['claim_type'] = @data['claim_status']
+      claim['data']['open'] = @data['phase_type'] != 'complete'
```

Comment:
Tested branch locally against staging and found a claim where the status is "Complete" but the "open" attribute is showing "true". I'm guessing the string used for comparison here should have first letter capitalized so it matches the actual phase/status of the claim?

URL for test: http://localhost:3000/services/claims/v1/claims/600098193